containers/Dockerfile.EESSI-client-pilot-centos7-ppc64le

```diff
@@ -0,0 +1,27 @@
+FROM docker.io/ppc64le/centos:7
+
+RUN yum install -y sudo vim openssh-clients cmake wget make unzip patch valgrind bzip2
+
+RUN yum install -y gcc gcc-c++ sqlite-devel python-devel libcap-devel libuuid-devel attr \
+ && yum install -y fuse-devel fuse3-devel zlib-devel openssl-devel
+
+# build CernVM-FS from source (no aarch64 Debian packages available)
+RUN  wget https://github.com/cvmfs/cvmfs/archive/cvmfs-2.7.5.tar.gz && \
```

Comment:
Should we have a variable for the CVMFS-version? 2.7.5 is repeated... a lot. Something like

```
ARG cvmfsversion=2.7.5 
```

And then interpolate it in the rest of the file?

Reply:
I copied most of it from this one:
https://github.com/EESSI/software-layer/blob/master/Dockerfile.fuse-overlay-debian10-aarch64
But I agree that it's much cleaner to use a variable for this. I'll take a look at that (and then add the same improvements to the one in software-layer, and update the version to 2.8.0).

Seems like there's also still some weird issue with the container (I'm getting weird errors for one repo) that needs to be solved.